utils/graph.h

```diff
@@ -9,13 +9,20 @@
 #include "utils/list.h"
 #include "utils/rbtree.h"
 
+struct graph_time_stat {
+	uint64_t sum;
+	uint64_t avg;
+	uint64_t min;
+	uint64_t max;
+};
+
 struct uftrace_graph_node {
 	uint64_t addr;
 	char *name;
 	int nr_edges;
 	int nr_calls;
-	uint64_t time;
-	uint64_t child_time;
+	struct graph_time_stat total_time;
+	struct graph_time_stat child_time;
```

Comment:
I don't think we need a stat for child time.  The current code is just to get the sum of self time easily.  But if we want to have min/max the logic has to change.

Reply:
@namhyung 
stat for child time Isn't it useful to store the average?